python/doc/usecases/use_case_fireSatellite.rst

```diff
@@ -0,0 +1,254 @@
+- :math:`t` : the thickness of solar panels, 0.005 m
+
+- :math:`n_{sa}` : the number of solar arrays, 3
+
+- :math:`\epsilon_{deg}` : the degradation in power production capability, 0.0375 percent per year
+
+- :math:`LT` : the lifetime of spacecraft, 15 years
+
+- :math:`r_{lw}` : the length to width ratio of solar array, 3
+
+- :math:`D` : the distance between panels, 2 m
+
+- :math:`I_{bodyX}` : the inertia of body, X axis, 6200 kg.m2
+
+- :math:`I_{bodyY}` : the inertia of body, Y axis, 6200 kg.m2
+
+- :math:`I_{bodyZ}` : the inertia of body, Z axis, 4700 kg.m2
+
+- :math:`\rho_{sa}` : the average mass density to arrays, 700 kg.m3
+
+- :math:`\eta` :  the power efficiency, 0.22
+
+- :math:`\phi_{target}` : the target diameter, 235000 m
+
+- :math:`R_E` : the Earth radius, 6378140 m
+
+We assume that the input variables are independent.
+
+The following figure depicts the interaction between the disciplines.    
+
+.. figure:: ../_static/N2firesat.png
+    :align: center 
+    :alt: use case geometry
+    :width: 80%
+
+The orbit discipline is defined as follows. First, the satellite velocity :math:`v` is computed from the Earth radius :math:`R_E` and the altitude :math:`H` .
+
+.. math::
+    v = \sqrt{\frac{\mu}{R_E+H}}
+
+with :math:`\mu` the Earth gravity constant. Then, the orbit period :math:`\Delta_{t_{orbit}}` is calculated,
+
+.. math::
+    \Delta_{t_{orbit}} = \frac{2\pi(R_E+H)}{v}
+
+The eclipse period :math:`\Delta_{t_{ecplise}}` and maximum slewing angle :math:`\theta_{slew}` are then computed,
+
+.. math::
+    \Delta_{t_{ecplise}} = \frac{\Delta_{t_{orbit}}}{\pi}\arcsin\left(\frac{R_E}{R_E+H}\right)
+
+.. math::
+    \theta_{slew} = \arctan\left(\frac{\sin\left(\frac{\phi_{target}}{R_E}\right)}{1-\cos\left(\frac{\phi_{target}}{R_E}\right)+\frac{H}{R_E}} \right)
+
+with :math:`\phi_{target}` the target diameter.
+
+The attitude and control discipline is governed by the following equations.
+
+.. math::
+    \tau_{tot} = \max(\tau_{slew},\tau_{dist})
+
+with
+
+.. math::
+    \tau_{slew} = \frac{4\theta_{slew}}{\Delta t_{slew}^2} I_{max}
+
+and 
+
+.. math::
+    \tau_{dist} = \sqrt{\tau_g^2 + \tau_{sp}^2 + \tau_m^2 + \tau_a^2}
+
+.. math::
+    \tau_{g} = \frac{3\mu}{2(R_E+H)^3}|I_{max}-I_{min}|\sin(2\theta)
+
+.. math::
+    \tau_{sp} = L_{sp}\frac{F_s}{C}A_s(1+q)\cos(i)
+
+.. math::
+    \tau_{m} = \frac{2 M R_D}{R_E+H)^3}
+
+.. math::
+    \tau_{a} = \frac{1}{2} L_{alpha} \rho C_d A v^2
+
+with :math:`\tau_{tot}` the total torque, :math:`\tau_{slew}` the slewing torque, :math:`\tau_{dist}` the disturbance torque, :math:`\tau_{g}` the gravity gradient torque, :math:`\tau_{sp}` the solar radiation torque, :math:`\tau_{m}` the magnetic filed interaction torque, :math:`\tau_{a}` the aerodynamic torque.
+
+The attitude control power :math:`P_{ACS}` is finally defined by
+
+.. math::
+    P_{ACS} = \tau_{tot}\omega_{max} + n P_{hold}
+
+The power discipline has 16 inputs and computes the total solar array size and total power by,
```

Comment:
Normally it's ok. 

I don't any idea of the reason of mac os test failure.

Reply:
That failure is unrelated to your PR. I guess scipy must have made a very recent change, and since our CI machines always pull the very latest version of all libraries...

Reply:
Indeed, since my comments several new pull requests have been created and all of them seem to have the same issue.